PURL-SPECIFICATION.rst

```diff
@@ -405,13 +413,17 @@ To parse a ``purl`` string in its components:
 - Split the ``remainder`` once from right on '/'
 
   - The left side is the ``remainder``
+  - Strip all leading characters (e.g., '/', '//' and so on)
+    from the right side
```

Comment:
If we split once from right, the right side will not have any `/` characters to strip.

Reply:
But splitting once could be understood as splitting on one slash